packages/diagrams/frontend/sirius-components-diagrams/src/renderer/palette/Palette.tsx

```diff
@@ -412,6 +419,19 @@ export const Palette = ({
     [editingContextId, diagramId, collapseExpandMutation]
   );
 
+  const handleDialogDescription = (tool: GQLSingleClickOnDiagramElementTool, diagramElementTargetObjectId: string) => {
+    const onConfirm = (variables: GQLToolVariable[]) => {
+      invokeSingleClickTool(tool, variables);
+    };
+    showDialog(
+      tool.dialog.dialogDescriptionType,
+      editingContextId,
+      tool.dialog.dialogDescriptionId,
+      diagramElementTargetObjectId,
```

Comment:
Don't pass around the `targetObjectId` of the element on which the end user has clicked.

Reply:
done